tests/test_start_date.py

```diff
@@ -202,40 +192,41 @@ def start_date_test(self):
                                      "Sync 2 start_date: {} ".format(self.START_DATE_2) +
                                      "Sync 2 record_count: {}".format(record_count_2))
 
+                # Verify 1st sync record count > 2nd sync record count for incremental streams
+                if replication_type == self.INCREMENTAL:
+                    self.assertGreater(replicated_row_count_1, replicated_row_count_2, msg="Expected less records on 2nd sync.")
+
+                elif replication_type != self.FULL:
+                    raise Exception("Expectations are set incorrectly. {} cannot have a "
+                                    "replication method of {}".format(stream, replication_type))
 
-                # Testing how FULL TABLE streams handle start date
-                if replication_type == self.FULL:
 
-                    # Verify that a bookmark doesn't exist for the stream.
-                    self.assertTrue(state_1.get(stream) is None,
-                                    msg="There should not be bookmark value for {}\n{}".format(stream, state_1.get(stream)))
-                    self.assertTrue(state_2.get(stream) is None,
-                                    msg="There should not be bookmark value for {}\n{}".format(stream, state_1.get(stream)))
+                # Skip the remaining assertions for inventories since it is append only
+                if stream == 'inventories':
+                    continue
 
 
-                # Testing how INCREMENTAL streams handle start date
-                elif replication_type == self.INCREMENTAL:
+                # BUG | https://stitchdata.atlassian.net/browse/SRCE-3681
+                # Skipping these two streams until BUG resolved.
+                # NOTE: we skip inventories ^ for a different reason leave that as is
+                if stream in {'roles', 'employees', 'locations'}: # TODO REMOVE
```

Comment:
Can you comment on the bug we need to remove this? I'm sure I'll forget otherwise